packages/validators/test/fixtures/v2023_09_28/experience.ts

```diff
@@ -0,0 +1,305 @@
+export const experience = {
+  metadata: {
+    tags: [],
+  },
+  sys: {
+    space: {
+      sys: {
+        type: 'Link',
+        linkType: 'Space',
+        id: 'tofsejyzyo24',
+      },
+    },
+    id: '64AN4WV1fN0myahCWSOUhq',
+    type: 'Entry',
+    createdAt: '2024-02-06T15:05:23.137Z',
+    updatedAt: '2024-02-06T15:09:54.939Z',
+    environment: {
+      sys: {
+        id: 'master',
+        type: 'Link',
+        linkType: 'Environment',
+      },
+    },
+    publishedVersion: 5,
+    publishedAt: '2024-02-06T15:09:19.540Z',
+    firstPublishedAt: '2024-02-06T15:09:19.540Z',
+    createdBy: {
+      sys: {
+        type: 'Link',
+        linkType: 'User',
+        id: '01pqG1mJhkXPX0GhF6jgGX',
+      },
+    },
+    updatedBy: {
+      sys: {
+        type: 'Link',
+        linkType: 'User',
+        id: '01pqG1mJhkXPX0GhF6jgGX',
+      },
+    },
+    publishedCounter: 1,
+    version: 7,
+    publishedBy: {
+      sys: {
+        type: 'Link',
+        linkType: 'User',
+        id: '01pqG1mJhkXPX0GhF6jgGX',
+      },
+    },
+    automationTags: [],
+    contentType: {
+      sys: {
+        type: 'Link',
+        linkType: 'ContentType',
+        id: 'hostedLayout',
+      },
+    },
+  },
+  fields: {
+    title: {
+      'en-US': 'Sofia test',
+    },
+    slug: {
+      'en-US': 'sofia-test',
+    },
+    componentTree: {
+      'en-US': {
+        breakpoints: [
+          {
+            id: 'desktop',
+            query: '*',
+            displayName: 'All Sizes',
+            previewSize: '100%',
+          },
+          {
+            id: 'tablet',
+            query: '<992px',
+            displayName: 'Tablet',
+            previewSize: '820px',
+          },
+          {
+            id: 'mobile',
+            query: '<576px',
+            displayName: 'Mobile',
+            previewSize: '390px',
+          },
+        ],
+        schemaVersion: '2023-09-28',
+        children: [
+          {
+            definitionId: 'contentful-container',
+            variables: {
+              cfVerticalAlignment: {
+                type: 'DesignValue',
+                valuesByBreakpoint: {
+                  desktop: 'center',
+                },
+              },
+              cfHorizontalAlignment: {
+                type: 'DesignValue',
+                valuesByBreakpoint: {
+                  desktop: 'center',
+                },
+              },
+              cfMargin: {
+                type: 'DesignValue',
+                valuesByBreakpoint: {
+                  desktop: '0 auto 0 auto',
```

Comment:
I didn't see any design value with token values, would be nice to have some in the fixtures, even though we don't do any special validation against those right now.